src/app-layout/visual-refresh-toolbar/drawer/styles.scss

```diff
@@ -52,7 +52,7 @@
       display: none;
 
       &.last-opened {
-        display: block;
+        display: grid;
```

Comment:
This is prone to breaking again.

The error proof implementation would be like this

```
@include mobile-only {
   &:not(.last-opened) {
      display: none;
   }
}
```

This way the code does what is intended and it does not depend on a random line of styles somewhere above